libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -552,17 +575,25 @@ if(LIBC_TYPES_HAS_FLOAT16)
     libc.src.math.canonicalizef16
     libc.src.math.ceilf16
     libc.src.math.copysignf16
+    libc.src.math.expf16
```

Comment:
We ran into an issue with `expf16` on the AArch64 buildbots that I couldn't reproduce on my AArch64 phone: https://github.com/llvm/llvm-project/pull/100632#issuecomment-2258772681.